### ISSUE TITLE
Include Latex3 warnings in log file summary

### ISSUE THREAD

**Issue by reitzig** [reitzig/texlogparser#3]
_From @samcarter8 on September 13, 2018 13:50_

Latex3 displays warning in a format like the following example:

```
*************************************************
* widows-and-orphans warning: "orphan-widow"
* 
* Orphan on page 1 (second column) and widow on page 2 (first column)
*************************************************
```
Would it be possible to show such warnings in the summary of the log file?

------

A short example that produces such a warning:

```
\documentclass{article}

\usepackage{lipsum}
\usepackage[check=warning]{widows-and-orphans}

\begin{document}
\lipsum[1-5]
test test testtest test testtest test testtest test testtest test testtest test testtest test testtest test test

test


\end{document}
```

_Copied from original issue: reitzig/ltx2any#113_

**Comment by reitzig** [comment]
Heh, how did I miss those? 

Will definitely include this pattern the next time I touch the library. Thanks for the report and the test case, perfect!

**Comment by reitzig** [comment]
For reference: the `widows-and-orphans` page is rather new; I had to run updates on TeXlive 2018 to get it.

**Comment by ghost** [comment]
Yes, it is rather new. I should have warned you about that, sorry!

**Comment by reitzig** [comment]
Implemented in v1.1.0.

**Comment by ghost** [comment]
Thanks a lot for fixing this!
